src/style.css

```diff
@@ -107,17 +109,37 @@ input[type=checkbox]:checked ~ {
 }
 
 .icon {
-  color: rgb(194, 181, 181);
+  color: #c2b5b5;
   margin-right: 10px;
 }
 
+.fa-trash-can:hover {
+  color: #c76161;
+}
+
+.fa-ellipsis-vertical {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+
+  /* background-color: #c76161; */
```

Comment:
- [ ] **Required:** Kindly remove all commented cod in your codebase. This is more professional and considered clean code.